src/components/marker-table/ContextMenu.js

```diff
@@ -84,6 +96,24 @@ class MarkersContextMenu extends PureComponent<Props> {
     });
   };
 
+  convertStackToString(stack: IndexIntoStackTable): string {
+    const { thread, implementationFilter } = this.props;
+
+    const callNodePath = filterCallNodePathByImplementation(
+      thread,
+      implementationFilter,
+      convertStackToCallNodePath(thread, stack)
+    );
+
+    const funcNamesAndOrigins = getFuncNamesAndOriginsForPath(
+      callNodePath,
+      thread
+    );
+    return funcNamesAndOrigins.map(
+      ({ funcName, origin }) => `${funcName} [${origin}]`
+    );
```

Comment:
You miss a last step here: convert the array returned by `map` to a string. I explained it in https://github.com/devtools-html/perf.html/issues/770#issuecomment-428511523 but maybe you missed it :) It's as simple as adding `.join('\n')` at the end.